### PR TITLE
Replace tpm2_print

### PR DESCRIPTION
**Pull request by stefanberger**
To simplify code and get rid of tpm2_print tool, implement a function that parses TPMS_ATTEST so far that it can extract TPMS_CLOCK_INFO.

### CODE REVIEW

File: keylime/tpm/tpm2_objects.py [hunk 1]
```diff
@@ -440,3 +441,26 @@ def ek_low_tpm2b_public_from_pubkey(pubkey: pubkey_type) -> bytes:
...
+def parse_tpms_clock_info(clock_info: bytes) -> Dict[str, int]:
+    (clock, resetCount, restartCount, safe) = struct.unpack(">QIIB", clock_info)
+    return {"clock": clock, "resetCount": resetCount, "restartCount": restartCount, "safe": safe}
+
+
+def get_tpms_attest_clock_info(tpms_attest: bytes) -> Dict[str, Any]:
+    (
+        magic,
+        _,
+    ) = struct.unpack_from(">IH", tpms_attest, 0)
```
Comment (aplanas) [inline keylime/tpm/tpm2_objects.py:455]: nit: but I wonder why black is doing this, when `magic, _ =  struct.unpack_from(">IH", tpms_attest, 0)` seems shorter that other lines

**Review by aplanas — APPROVED** [review]
Very cool one.

File: keylime/tpm/tpm2_objects.py (hunk 1 above)
Comment (stefanberger) [reply keylime/tpm/tpm2_objects.py:455]: don't try to overrule `black` ...

Comment (aplanas) [reply keylime/tpm/tpm2_objects.py:455]: yeah, it does not matter

Comment (stefanberger) [reply keylime/tpm/tpm2_objects.py:455]: I don't know why black did this...

Comment (aplanas) [reply keylime/tpm/tpm2_objects.py]: there is a chance that it is a bug somewhere, here it is working tho:

```
# pip list | grep black
black           23.3.0

# cat kk.py
import struct


def get_tpms_attest_clock_info(tpms_attest):
    magic, _ = struct.unpack_from(">IH", tpms_attest, 0)

# black --diff kk.py 
All done! ✨ 🍰 ✨1 file would be left unchanged.
```

**Review by THS-on — APPROVED** [review]

**Pull request merged**